scripts/lint

```diff
@@ -1,14 +1,14 @@
 #!/bin/sh -e
 
-export PREFIX=""
+PACKAGE="httpcore"
+
 if [ -d 'venv' ] ; then
-    export PREFIX="venv/bin/"
+    PREFIX="venv/bin/"
+else
+    PREFIX=""
 fi
-export SOURCE_FILES="httpcore tests"
 
 set -x
 
-${PREFIX}autoflake --in-place --recursive $SOURCE_FILES
-${PREFIX}isort --project=httpcore --recursive --apply $SOURCE_FILES
-${PREFIX}black --exclude 'httpcore/_sync/.*' --exclude 'tests/sync_tests/.*' --target-version=py36 $SOURCE_FILES
+${PREFIX}black --exclude '(httpcore/_sync|tests/sync_tests)/.*' --target-version=py37 ${PACKAGE} tests
```

Comment:
Thoughts on keeping `autoflake`? If we're adding `flake8` to `check` it's definitely nice to have a quick way to fix things when they're broken.